Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/WebViewSamplePage.xaml.cs

```diff
@@ -2,8 +2,8 @@
 
 namespace Uno.Gallery.Views.Samples
 {
-	// There is no WebView implementation in WebAssembly and MacOS
-#if !__WASM__ && !__MACOS__
+	// There is no WebView implementation in WebAssembly
+#if !__WASM__
```

Comment:
Is it activated for SKIA?

Reply:
@carldebilly It should not be 😮 ! I disabled the sample there.